src/index.ts

```diff
@@ -33,19 +33,17 @@ async function fetchPrice(name: string): Promise<number> {
 }
 
 async function fetchFtxPrice(name: string): Promise<number> {
+    name = name.toUpperCase()
     log(`[FTX] fetching price of ${name}`)
     const url = `${FTX_ENDPOINT}/${name}/USD`
     const resp = await fetch(url, { method: 'GET', timeout: 10 * 1000 })
-    const price = (await resp.json()).result.last
+    const price = (await resp.json()).result?.last
```

Comment:
should we throw an error if the price is `undefined`? Otherwise undefined might be saved into markets and returned.